src/go.mod

```diff
@@ -14,5 +14,46 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 )
 
+require (
+	github.com/Microsoft/go-winio v0.4.16 // indirect
```

Comment:
Just curious, did these get added automatically when building for 1.17 or did you have to manually add these in?